app-games/src/main/java/com/aptoide/android/aptoidegames/feature_apps/presentation/CarouselLargeAppView.kt

```diff
@@ -145,6 +146,13 @@ private fun CarouselLargeAppView(
   app: App,
   onClick: () -> Unit,
 ) {
+  val downloadUiState = rememberDownloadState(app = app)
+
+  val progressShowing =
```

Comment:
the name is a bit miss leading because the error state should have only one line and it doesn't show progress. maybe this should be called "appNameMaxLines" and it would be passed to the app name